doc/analysis/index.rst

```diff
@@ -15,7 +15,7 @@
 .. _overview:
 
 An overview of the analysis module
-===========================================
```

Comment:
I suspect there are more like this but I decided not to start looking to give my inner marie kondo a break.